graviti/manager/dataset.py

```diff
@@ -249,7 +249,13 @@ def commit(
                 "Please checkout a branch first"
             )
         if not head.operations:
-            raise StatusError("It is not allowed to commit a dataset without any modifications")
+            without_modifications = True
```

Comment:
is it possible to describe the issue more detailed in the commit message:
here is a reference:
```
fix(manager): fix dataset.commit failed when the sheet list is unchanged
```